man/man8/zpool.8

```diff
@@ -1550,14 +1550,16 @@ Run a script (or scripts) on each vdev and include the output in zpool iostat
 .sp
 The \fB-c\fR option allows you to run script(s) for each vdev and display the
 output in zpool iostat.  For security reasons, a user can only execute scripts
-found in the /<etc>/zfs/zpool.d directory as an unprivileged user.  However, a
-privileged user can run \fB-c\fR if they have the ZPOOL_SCRIPTS_AS_ROOT
+as an unprivileged user. By default, a user may run a script from <home>/.zpool.d,
+/<etc>/zfs/zpool.d, or provide a full path to a script. The default search path
```

Comment:
I currently have it as `/<etc>/zfs/zpool.d`. It doesn't seem to mess up how man displays it.